web/frontend/src/components/slam/index.svelte

```diff
@@ -243,15 +343,41 @@ useDisconnect(() => {
     on:click={handleStopMoveClick}
     on:keydown={handleStopMoveClick}
   />
-  <div class="flex flex-wrap gap-4 border border-t-0 border-medium sm:flex-nowrap">
-    <div class="flex min-w-fit flex-col gap-4 p-4">
-      <div class="float-left pb-4">
-        <div>
-          <p class="mb-1 font-bold text-gray-800">
-            Map
-          </p>
-          <div class="flex items-end gap-2 w-64">
-            <div class="relative">
+  <div
+    class="flex flex-wrap gap-4 border border-t-0 border-medium sm:flex-nowrap"
+  >
+    <div class="flex min-w-fit flex-col gap-4 p-4 pr-0">
+      <div class="pb-4 flex flex-col gap-6">
+        {#if overrides?.isMapping && overrides?.mappingDetails}
+          <header class="flex flex-col text-xs justify-between gap-3">
+            <div class="flex flex-col">
+              <span class="font-bold text-gray-800">Mapping mode</span>
+              <span class="capitalize text-subtle-2"
+                >{overrides.mappingDetails.mode}</span
+              >
+            </div>
+            <div class="flex gap-8">
+              {#if overrides.mappingDetails.name}
+                <div class="flex flex-col">
+                  <span class="font-bold text-gray-800">Map name</span>
+                  <span class="capitalize text-subtle-2"
+                    >{overrides.mappingDetails.name}</span
+                  >
+                </div>
+              {/if}
+              {#if overrides.mappingDetails.version}
+                <div class="flex flex-col">
+                  <span class="font-bold text-gray-800">Version</span>
+                  <span class="capitalize text-subtle-2"
+                    >{overrides.mappingDetails.version}</span
+                  >
+                </div>
+              {/if}
+            </div>
+          </header>
+        {/if}
+        <div class="flex items-end gap-2 min-w-fit">
+          <div class="relative">
               <p class="mb-1 text-xs text-gray-500">
                 Refresh frequency
```

Comment:
made a ticket to verify what refresh rates we should support for cloud slam https://viam.atlassian.net/browse/RSDK-4174